swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition = 0):
+        super().__init__(category="Clothing", condition = condition)
+    
+    def __str__(self):
+        return "The finest clothing you could wear."
```

Comment:
Great use of the parent/super constructor Mary!! Passing in category as a default argument prevents users from changing the category type of a Clothing instance. Good work!